script.js

```diff
@@ -0,0 +1,76 @@
+document.addEventListener('DOMContentLoaded', function () {
+  const awesomeList = document.getElementById('awesomeList');
+  const textfieldTitle = document.getElementById('textfieldTitle');
+  const textfieldAuthor = document.getElementById('textfieldAuthor');
+  const btnAdd = document.getElementById('btnAdd');
+
+  function createBookItem(title, author) {
+    const bookItem = document.createElement('div');
+    const bookTitle = document.createElement('span');
+    const br = document.createElement('br');
+    const bookAuthor = document.createElement('span');
+    const removeBtn = document.createElement('button');
+    const hr = document.createElement('hr');
+    const li = document.createElement('br');
+    const lii = document.createElement('br');
+
+    bookTitle.textContent = title;
+    bookAuthor.textContent = author;
+    removeBtn.textContent = 'Remove';
+
+    removeBtn.addEventListener('click', function () {
+      bookItem.remove();
+      saveBooks();
+    });
```

Comment:
- [ ] Asper this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md#:~:text=Create%20a%20function%20to%20remove%20a%20book), Please create a function to remove a book from the collection (hint: you can use the [array filter()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) method) then you can call it here in the remove button event listener.